projects/storefrontlib/src/lib/ui/components/form-components/item-counter/item-counter.component.spec.ts

```diff
@@ -20,7 +20,7 @@ const testData = [
   { incomingValue: 6, adjustedValue: 5, isOutOfRange: true }
 ];
 
-describe('ItemCounterComponent', () => {
+fdescribe('ItemCounterComponent', () => {
   let itemCounterComponent: ItemCounterComponent;
   let fixture: ComponentFixture<ItemCounterComponent>;
 
```

Comment:
Cant write this anywhere else. Add a test to make sure something happens (warning) id the value is invalid.